app/controllers/users_controller.rb

```diff
@@ -9,8 +9,8 @@ def create
       redirect_to @user
       flash[:success] = "Logged in as #{@user.username}"
     else
-      flash.now[:error] = 'Username and password are required.'
-      render :new
+      flash[:error] = 'Username and password are required.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.